src/components/Button/Button.js

```diff
@@ -23,48 +23,47 @@ const propTypes = {
   onClick: PropTypes.func.isRequired,
 };
 
+const NOOP = () => {};
+
 const Button = props => {
   const [isElementDisabled, customOverrides] = useSelector(state => [
```

Comment:
what is different between the `disabled` props and `isElementDisabled`?

I'm guessing `isElementDisabled` takes higher priority over "disabled" ?

Reply:
`isElementDisabled` will be true if this element is disabled by `instance.disableElements` so this will mean that we don't render this 'button' at all.

When `disable` is true we just make it not clickable.

I can see that the name is confusing. Do you want to use a variable name other than `isElementDisabled` just in this component to make the difference a bit clear? Let me know what names you want to use.

Unfortunately I don't think we can rename `instance.disableElements` to something else for now because this API(or the `disabledElements` constructor option) is used a lot...

Reply:
Seeing how people fork this repo and may potentially touch this code.
Lets use a variable name to make things more clear.

Lets call it ` hideElement` instead of `isElementDisabled` ?

Reply:
i see that its renamed to `removeElement`